spread/report.go

```diff
@@ -0,0 +1,14 @@
+package spread
+
+type Report interface {
```

Comment:
interface names are usually verbs if possible, so `Reporter` ? does it need to be exported?